composer/utils/misc.py

```diff
@@ -62,3 +65,14 @@ def model_eval_mode(model: torch.nn.Module):
         yield
     finally:
         model.train(mode=is_training)
+
+
+def is_torch_xla_installed() -> bool:
+    """Whether torch_xla is installed in the current environment."""
+    try:
+        import torch_xla.core.xla_model as xm
+        del xm
+    except ImportError:
+        return False
+    else:
+        return True
```

Comment:
```suggestion
def is_torch_xla_installed() -> bool:
    """Whether torch_xla is installed in the current environment."""
    try:
        import torch_xla.core.xla_model as xm
        del xm
    except ModuleNotFoundError:
        return False
    else:
        return True
```
nit: if there's an error where module is found but it fails at some other step of importing, do we want to raise this to user? my instinct is yes